test.js

```diff
@@ -74,6 +74,33 @@ test('excludes and includes options', function (t) {
 	t.end();
 });
 
+test('shortForms options', function (t) {
+	var actual = dargs({a: 'foo', file: 'test'}, {shortForms: {file: 'f'}});
+	var expected = [
+		'--a=foo',
```

Comment:
This should be with one dash, as it should be possible to pass short forms directly too.